src/schema/vivliostyle/vivliostyleConfig.schema.json

```
@@ -166,33 +206,70 @@
           "minLength": 1
         },
         "timeout": {
+          "description": "Timeout limit for waiting Vivliostyle process [60s]",
```

timeout のデフォルトは120秒です。単位はコマンドラインオプションでは秒単位ですが、configでは1/1000秒単位です。
```suggestion
          "description": "Timeout limit for waiting Vivliostyle process [120000]",
```